backend/app/api/endpoints/users.py

```
@@ -877,13 +877,15 @@ def get_team_info(team_id: int, is_system: bool) -> Optional[QuickAccessTeam]:
 
 @router.get("/recent-teams", response_model=list[QuickAccessTeam])
 async def get_user_recent_teams(
+    is_code: bool = False,
     db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)